tests/ops/fixtures/application_fixtures.py

```diff
@@ -592,6 +590,291 @@ def erasure_policy_two_rules(
         pass
 
 
+@pytest.fixture(scope="function")
+def erasure_policy_complete_mask(
```

Comment:
Let's move this to `saas_example_fixtures.py` since we'll only use this in the SaaS integration tests